features/step_definitions/landing_page_steps.rb

```diff
@@ -0,0 +1,20 @@
+When(/^I visit the LandingPage$/) do
+  landing_page.load
+end
+
+Then(/^I want to be able to input my age$/) do
+  expect(page).to have_css('#your_details_form_age')
+end
+
+And(/^I want to be able to input my gender$/) do
+  expect(page).to have_css('#your_details_form_gender')
+end
+
+And(/^I want to be able to input my salary$/) do
+  expect(page).to have_css('#your_details_form_salary')
```

Comment:
I kind of agree if this is actually what the Scenario is testing. 
i.e. is it a Feature + scenario testing that you can input valid data?